packages/dom/src/dom/is-entirely-selected.js

```diff
@@ -1,18 +1,23 @@
 /**
- * External dependencies
+ * Internal dependencies
```

Comment:
If we keep this block, let's remove the other `Internal dependencies` one several lines below.